### PR TITLE
Autocomplete for default config fix #222

### LINKED ISSUE #222: Autocompletion for default configuration

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: client/src/main/scala/com.olegych.scastie.client/components/Editor.scala [hunk 1]
```diff
@@ -460,6 +477,70 @@ object Editor {
...
+            )
+          },
+          js.Dictionary(
+            "container" -> dom.document.querySelector(".CodeMirror"),
+            "alignWithWord" -> true,
+            "completeSingle" -> (next.completions.length == 1)
+          )
+        )
+
+        // FIXME: find another way to do it
```
Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/components/Editor.scala]: The way to do this is to apply the delta between `current: Option[Editor]` and `next: Editor`. You will find elements to add and elements to remove. More simply you can set the list using `next: Editor` if `current != Some(next)`

**Comment by MasseGuillaume** [comment]
Excellent work @dimart!

File: client/src/main/scala/com.olegych.scastie.client/components/Editor.scala (hunk 1 above)
Comment (dimart) [inline client/src/main/scala/com.olegych.scastie.client/components/Editor.scala]: But what if user made the same request twice?
Then the delta would be zero, but the user expects to see autocomplete items again.

Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/components/Editor.scala]: good point. Then you want to set a stat like `isAutocompleteOpen: Boolean`

**Comment by dimart** [comment]
Thanks @MasseGuillaume!

**Pull request merged**